Backend/mentii/user_ctrl.py

```diff
@@ -7,6 +7,10 @@
 import uuid
 import hashlib
 
+import sys
+from os import path
+sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
```

Comment:
Is there no way to fix this? It is being added with this commit, that's why I bring it up.